components/SearchInput.js

```diff
@@ -0,0 +1,27 @@
+const React = require('react');
+
+const SearchInput = ({
+  label,
+  size,
+  handleChange,
+  handleSubmit
+}) => (
+  <form onSubmit={handleSubmit}>
+    <label htmlFor='search-input'>{label}</label>
+    <input id='search-input' type="text" style={{width: size}} onChange={handleChange} />
+    <button type="submit">Submit</button> 
+  </form>
+);
+SearchInput.defaultProps = {
```

Comment:
I think we can use PropTypes for `handleChange` and `handleSubmit` to make sure these props are defined